File: co.edu.uniandes.fixml.toJava/test/text2.xml/Pty.java
package text2.xml;



// Class for tag Pty
public class Pty {

	String ID;
	String R;
	
	List<Sub> Subs = new ArrayList<Sub>();
	
	// default constructor
	public Pty ( String ID, String R ) {
	
		this.ID = ID;
		this.R = R;
		
	}
	
	public addSub( Sub child ) {
		this.Subs.add( child );
	}
	
	
	
} 

File: co.edu.uniandes.fixml.toC++/test/text2.xml/Pty.cpp
package text2.xml;

public class Pty {

	private:
  	String ID;
  	String R;

	Sub* Sub_1; 
	

 	public: 

  	// default constructor
	public Pty () {
		ID = "OCC" ;
		R = "21" ;
		
			Sub_obj = new Sub();
  	
  	}
  	
  	
  	
  
  	// by parameters
  	public Pty (
	  String ID_, String R_, 
			Sub* Sub_1,
  	){
	  
	  ID = ID_;
		R = R_;
	  
		this.Sub_1 = Sub_1;
	
  }
} 